pom.xml

```diff
@@ -233,7 +233,7 @@ limitations under the License.
                     <execution>
                         <phase>process-sources</phase>
                         <goals>
-                            <goal>check</goal>
```

Comment:
please leave this as check, I don't like the idea of maven updating the source files w/o me asking it to

Reply:
Reverted